ci_tools/bumpversion.sh

```diff
@@ -0,0 +1,5 @@
+bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
+git config --local user.email "engineering@finbourne.com"
+git config --local user.name "finbourne-bot-public"
+git push https://finbourne-bot-public:${GH_TOKEN}@github.com/finbourne/lusid-python-tools.git HEAD:$TRAVIS_BRANCH
```

Comment:
What does "HEAD" do here? Is it required?

Reply:
Making a commit directly to the branch in Travis causes the HEAD to detach. See logs below. Therefore it seems we need to push this commit back to HEAD again. 


Logs from Travis
----------------------------------------------------------
[0Ktravis_time:start:1c1aaaf4
[0K$ bash ci_tools/new_commit_with_black.sh
[1mreformatted /home/travis/build/slemasne/repo-test/black_test.py[0m
[1mAll done! âœ¨ ðŸ° âœ¨[0m
[1m1 file reformatted[0m, 6 files left unchanged.[0m
**[detached HEAD 7cddf67] black code formatting**
 1 file changed, 1 insertion(+), 4 deletions(-)
Everything up-to-date